dirty_cat/test/test_fuzzy_join.py

```diff
@@ -67,57 +69,72 @@ def test_fuzzy_join(analyzer, match_type, return_distance=True):
         }
     )
 
+    # Check correct shapes of outputs:
     teams_joined, dist1 = fuzzy_join(
         teams1,
         teams2,
         on=["basketball_teams", "teams_basketball"],
-        return_distance=return_distance,
+        return_distance=True,
         analyzer=analyzer, match_type=match_type,
         match_threshold=0.1
     )
-    # Check correct shapes of outputs
     assert teams_joined.shape == (9, 2)
     assert dist1.shape == (9, 1)
     assert (teams_joined == ground_truth).all()[1]
+    # And on the other way around:
     teams_joined_2, dist2 = fuzzy_join(
         teams2,
         teams1,
         on=["teams_basketball", "basketball_teams"],
-        return_distance=return_distance,
+        return_distance=True,
         analyzer=analyzer, match_type=match_type,
         match_threshold=0.1
     )
     # Joining is always done on the left table and thus takes it shape:
     assert teams_joined_2.shape == (10, 2)
     assert dist2.shape == (10, 1)
 
+    # Check invariability of joining:
     teams_joined_3 = fuzzy_join(
         teams2,
         teams1,
         on=["teams_basketball", "basketball_teams"],
-        return_distance=False,
         analyzer=analyzer, match_type=match_type,
         match_threshold=0.1
     )
-    # Check invariability of joining:
     pd.testing.assert_frame_equal(teams_joined_2, teams_joined_3)
 
+    # Check keep argument: 
+    teams_kept = fuzzy_join(
+        teams1,
+        teams2,
+        on=["basketball_teams", "teams_basketball"],
+        analyzer=analyzer, match_type=match_type,
+        match_threshold=0.1,
+        keep = keep
+    )
+    # If left, should be identical to the left table column:
+    if keep=='left':
+        pd.testing.assert_frame_equal(teams_kept, teams1)
+    # If right, should take the shape of the left table column:
+    if keep=='right':
+        assert teams_kept.shape == teams1.shape
```

Comment:
I don't really like these comments, they explain "what" but not "why". 
They are redundant with the code (they can be guessed from it without too much difficulty), however people reading this might not necessarily understand why we should check that.
As comparison, a comment above does this well:
```python
    # Joining is always done on the left table and thus takes it shape:
    assert teams_joined_2.shape == (10, 2)
    assert dist2.shape == (10, 1)
```
Also, I feel like the pre-commit hooks are not installed on your env, there should be spaces before and after the double equal :)
[Please check you do ;)](https://github.com/dirty-cat/dirty_cat/blob/master/CONTRIBUTING.md)